tools/runners/run-test-suite-test262.py

```diff
@@ -189,6 +197,10 @@ def main(args):
     if 'excludelist_path' in args and args.mode == 'default':
         test262_command.extend(['--exclude-list', args.excludelist_path])
 
+    if args.test262_test_list:
+        for test in args.test262_test_list.split(','):
```

Comment:
in python you can do: `test262_command.extend(args.test262_test_list.split(','))` to achieve the same thing.